### ISSUE TITLE
[FEAT] Native support for sharing password via QR code

### ISSUE THREAD

**Issue by mrkobel** [PhilippC/keepass2android#2402]
I love your app! Saves me so much time. But there is one feature I really wish I had. Tha available plugin doesn't work on my Samsung S21 (not compatible with my android version).

## Summary
I keep my database on my mobile device. I sometimes need to sign in to my accounts from a company computer, where I don't want to keep my database / can't install KeepassXC. My passwords are usually autogenerated, so it's cumbersome to type them in manually. The workaround I came up with is to use an external app to generate a QR code containing my password and use the Windows Camera app to scan it using my notebooks built-in camera. It's a bit cumbersome, because I need to first copy my password, paste it in another app and share the text via QR code generator from there. Would be great if KeepassXC mobile app could do this for me. 

## Examples
I need to sign in to my google account on a trusted computer, which I do not own. 
I locate my password in KeepassXC app on my mobile, select password and click "generate QR code"
I scan the code with built-in windows 10 camera app, copy the password and paste it into the sign-in form. 

thanks,
Miron

**Comment by NumbGnat** [comment]
Somewhere within all of these issues, someone referenced some sort of Bluetooth keyboard typing interface, that you could plug into your computer. It would simulate keystrokes and there was a way to link that device to your Android phone so that your Android could type keystrokes into your computer.

I'm sorry I don't have more time to research that and find the link to what I was talking about, but maybe you can poke around a bit and that might be a good solution for you.

**Comment by fpasqua75** [comment]
Yes, it would be really useful to display username, password and TOTP values as barcodes, to read them i.e. with an USB scanner...
It's complicated to input manually when you have long random passwords.

**Comment by timi35** [comment]
Same here, the QR barcode function works great on Keepass-Windows. Please add it to your Android version.
